src/main/java/roomescape/exception/RoomescapeExceptionHandler.java

```diff
@@ -27,7 +29,15 @@ public ErrorResponse handle(HttpMessageNotReadableException e) {
     @ExceptionHandler(RoomescapeException.class)
     public ErrorResponse handle(RoomescapeException e) {
         logger.error(e.getLogMessage());
-        return ErrorResponse.builder(e, e.getHttpStatus(), e.getMessage())
+        return ErrorResponse.builder(e, e.getExceptionType().getStatus(), e.getMessage())
+                .build();
+    }
+
+    @ExceptionHandler(PaymentException.class)
+    public ErrorResponse handle(PaymentException e) {
+        PaymentExceptionResponse paymentExceptionResponse = e.getPaymentExceptionResponse();
+        logger.error("{}{}{}", paymentExceptionResponse.getErrorCode(), System.lineSeparator(), paymentExceptionResponse.getMessage());
```

Comment:
```suggestion
        logger.error("{} {}", paymentExceptionResponse.getErrorCode(), paymentExceptionResponse.getMessage());
```
와 무엇이 다를까요?
+ `{} {}`보다는 좀 더 명확하게 각 값이 뭘 뜻하는지를 표현하는게 더 좋아보이네요.

Reply:
각 `log`를 직접 출력해봤을 때 아래와 같은 차이가 존재하였습니다.
보다 명확하게 어떤 값을 `log`로 표현하고자 했는지 나타내는 방향으로 리팩토링 하였습니다!

`logger.error("{}{}{}", paymentExceptionResponse.getErrorCode(), System.lineSeparator(), `
``` 
paymentExceptionResponse.getMessage());
2024-06-04T15:10:49.402+09:00 ERROR 54978 --- [io-8080-exec-10] r.exception.RoomescapeExceptionHandler   : UNAUTHORIZED_KEY
서버 내부에서 결제 연동 상의 문제가 발생하였습니다. 잠시 후에 다시 시도해주세요
```

`logger.error("{} {}", paymentExceptionResponse.getErrorCode(), paymentExceptionResponse.getMessage());`
```
2024-06-04T15:10:49.403+09:00 ERROR 54978 --- [io-8080-exec-10] r.exception.RoomescapeExceptionHandler   : UNAUTHORIZED_KEY 서버 내부에서 결제 연동 상의 문제가 발생하였습니다. 잠시 후에 다시 시도해주세요
```